### DISCUSSION TITLE
[功能请求] 请不要放弃三级阴影 三级多好看 这也是 antd 与众不同的地方

### DISCUSSION THREAD

**Discussion by l-7-l** [ant-design/ant-design#47078] [General]
### What problem does this feature solve?

手动控制阴影选择

### What does the proposed API look like?

token: {
    shadownKind:  2 | 3,
 }

**Comment by l-7-l** [comment]
我记得以前能交互的组件 比如 button 层次感很强的 和波纹一样 现在再努力一点就要糊成边框了

**Comment by afc163** [comment]
三级阴影是什么意思？

**Reply by yoyo837** [reply]
点击时的wave吧，由深到浅三层颜色。

**Reply by li-jia-nan** [reply]
wave 阴影其实没变的，只是以前点击的时候会取消上次的 wave，现在删了这个逻辑，每次点击 wave 都会叠加

**Comment by l-7-l** [comment]
> 点击时的wave吧，由深到浅三层颜色。

对的，很有层次感 以前的 wave 交互风格可以说是独一无二的

**Reply by afc163** [reply]
@l-7-l 来个对比的 codesandbox 重现？我其实没 get 到有啥区别

**Reply by l-7-l** [reply]
@afc163 眼花了 我怎么看着感觉区别也不大了 https://1x.ant.design/components/button/
